e2etest/e2etest_suite_test.go

```diff
@@ -50,6 +50,15 @@ func TestMain(m *testing.M) {
 	os.Exit(m.Run())
 }
 
+// use ephemeral port (tcp/8080) for pod, instead of well-known port (tcp/80)
```

Comment:
I think the comment is incorrect as the port is not 8080 (it's configurable). Also, it is not the ephemeral port.

Reply:
port 8080 could be ephemeral port based on RFC6056. https://datatracker.ietf.org/doc/html/rfc6056#section-2.1

I changed the comment.

Reply:
Ephemeral ports usually refer to the source port of a connection to a server (i.e. the client side port). Perhaps in this context (pod backing a service) this term is also used so its a bit confusing. How about:

"use configured port for pod instead of default port (tcp/80)"

Although, I am now also starting to doubt I understand the problem you are solving. Could you explain in the commit message why you need this PR?